conan/cli/formatters/audit/vulnerabilities.py

```diff
@@ -12,10 +12,9 @@
 }
 
 
-def text_vuln_formatter(result):
+def text_vuln_formatter(result: dict):
     from conan.api.output import cli_out_write, Color
-
-    response, errors_in_response = result
+    errors_in_response = "conan_error" in result
```

Comment:
but doing this you are ignoring the `errors_in_response` passed by the provider?

Reply:
Yes. Is this error coming the server directly or is something from client side? Asking if its something that we can treat in the data directly, without this extra flag. Otherwise, we would need to change _dispatch_errors, as commented here: https://github.com/conan-io/conan/pull/18160#discussion_r2055452686

Reply:
the reason of this is that if for example, you are requesting 30 cve's but it fails in the 25 because you run out of rate limit or anything, the formatter is able to print the information for those 25 and fail after that. 
I think this part may need a refactor but that's how it's working right now.